### ISSUE TITLE
[Parameter Request] Codeunit 7322 "Create Inventory Pick/Movement" - Function "CreatePickOrMoveLine" - New Parameter "sender" to OnCreatePickOrMoveLineOnAfterFindTakeQtyForBinCodeSourceLine

### ISSUE THREAD

**Issue by Sv1N2S** [microsoft/ALAppExtensions#22649]
old:

```
    [IntegrationEvent(false, false)]
    local procedure OnCreatePickOrMoveLineOnAfterFindTakeQtyForBinCodeSourceLine(var NewWarehouseActivityLine: Record "Warehouse Activity Line"; WhseItemTrackingSetup: Record "Item Tracking Setup"; var RemQtyToPickBase: Decimal; WarehouseActivityHeader: Record "Warehouse Activity Header")
    begin
    end;
```

new:

```
    [IntegrationEvent(true, false)]
    local procedure OnCreatePickOrMoveLineOnAfterFindTakeQtyForBinCodeSourceLine(var NewWarehouseActivityLine: Record "Warehouse Activity Line"; WhseItemTrackingSetup: Record "Item Tracking Setup"; var RemQtyToPickBase: Decimal; WarehouseActivityHeader: Record "Warehouse Activity Header")
    begin
    end;
```

**Comment by NadiaMelnyk** [comment]
@AlexanderYakunin 
Hello. Have you had a chance to review this request? Are there any updates on this?

**Comment by AleksandricMarko** [comment]
Include sender = true is implemented from BC 22.2
